README.md

```diff
@@ -120,25 +125,30 @@ $ git clone https://github.com/softdevteam/krun.git
 $ cd krun
 ```
 
-## Step 3: set `JAVA_HOME`
+## Step 3: Build Krun
+
+The Krun Makefile honours the standard variables: `CC`, `CPPFLAGS`, `CFLAGS`
+and `LDFLAGS`. For example, if you wish to use `clang` rather than `gcc` you
+can append `CC=/bin/clang` to the `make` command here:
```

Comment:
We can't protect people from everything, and trying to do so can sometimes make things worse.

That said, I don't care that much in this case (personally, I think we can assume that people know what to do with the `CC` variable. I don't even think we need to give an example).